proto/spire/api/registration/registration.proto

```diff
@@ -99,6 +99,48 @@ message EvictAgentResponse {
     spire.common.AttestedNode node = 1;
 }
 
+message MintX509SVIDRequest {
+    // SPIFFE ID of the X509-SVID
+    string spiffe_id = 1;
+
+    // ASN.1 encoded CSR. No fields from the CSR are used. It is only used
```

Comment:
saying that "no fields are used" is just slightly underspecified for me, since the CSR doesn't just prove ownership of the public key, it also conveys that very public key

maybe something like:
```
// ASN.1 DER encoded CSR. The only field retrieved from the CSR is the public key.
// All other fields are ignored. The CSR is used to convey proof of ownership of the public key.
```

Reply:
👍